templates/login.hamlet

```diff
@@ -1,12 +1,3 @@
-$if development
```

Comment:
Oh, I meant to mention that... Since the new scaffold doesn't have `development`, I had to do something here. I had plans to remove this for reasons I'll go into, so I just did it now.

Basically, the Dummy Login is useful only in test. It allows us to authenticate as a given user in a test by POSTing directly to the dummy auth route (`TestImport.authenticateAs`). It does not go by way of this form, so removing it doesn't impact that.

In development, it doesn't work anyway for the reasons stated here in #154.

Even if I fixed that issue, this would still not be useful because the user must already exist for the Dummy login to work. For the user to already exist, you must've set up GitHub auth. If you've set up GitHub auth, you might as well use the actual GitHub login.

The only place the Dummy login would be useful is if you were trying to develop without an internet connection. I don't see that as a reason to keep it.